dask/dataframe/core.py

```diff
@@ -413,9 +413,13 @@ def _repr_divisions(self):
 
     def __repr__(self):
         data = self._repr_data().to_string(max_rows=5, show_dimensions=False)
-        return """Dask {klass} Structure:
+        _str_fmt = """Dask {klass} Structure:
 {data}
-Dask Name: {name}, {task} tasks""".format(
+Dask Name: {name}, {task} tasks"""
+        if "Empty" in data:
```

Comment:
May have to be more sophisticated here. A user could have a column "Empty", which I think would put us here when we don't want to.

I think `len(self.columns)` should be sufficient.